machina/templates/machina/partials/posted_by.html

```diff
@@ -0,0 +1,33 @@
+{% load forum_member_tags %}
```

Comment:
I think we could probably improve how this partial works so that it - kindof - keep working like things were working previously.

For example why not:

* have a partial that either works from a single `post` variable or `topic` variable (depending on whether you want to display the poster info for the post or the topic itself) - this would allow to avoid passing `poster`, `username` and `creation_date` systematically in each partial use (which would be less error prone in my opinion)
* then in order to control whether the creation date has to be displayed, you could use an extra `show_creation_date` variable (True / False)
* `no_trans` would be replaced by something more explicit like `show_by_prefix`
* `username_highlighted` would be replaced by `highlight_username` (True / False) to improve consistency

What do you think?

Reply:
Yes, definitely, I think it's way simpler this way. I'll work around it and suggest it as a fix.
thank you for you feedback

Reply:
I've commited with a fixup, I had forgotten to add a file with the search in the previous commit which was another case. 
I don't know what to think of this new version. I had to make more cases to be able to read username as it is not specified anymore.